src/mozmlops/templates/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1
+
+# Comments are provided throughout this file to help you get started.
+# If you need more help, visit the Dockerfile reference guide at
+# https://docs.docker.com/go/dockerfile-reference/
+
+ARG TAG=3.10.11-bullseye
+FROM --platform=linux/amd64 python:${TAG} as base
```

Comment:
I was using TAG because I was going back and forth with saving to figure out which specific python image tag to pull, and so having that in its own variable made sense at the time. I'm ok with either approach though! We don't use that anywhere other than the line right below the declaration.